### DISCUSSION TITLE
'Permission Denied' Error while training spaCy 3.0 model using project.yml file

### DISCUSSION THREAD

**Discussion by palitr** [explosion/spaCy#8588] [Help: Coding & Implementations] [answered]
I am building a ner model using spaCy 3.0 for a client. All the specifications related to the model execution happens through project.yml file which exists in my project root folder. When I execute the model from CLI using the command `python -m spacy project run rerun-training --force`, all the steps run but training step fails with the error 'permission denied in training/model-best folder'. 
Btw, rerun-training is one of the workflows in thr project.yml file. I am working in a client environment where I cannot launch anaconda prompt in administrator mode. Is there a way to solve this problem?

**Comment by palitr** [comment, marked as answer]
Screenshot of the error
![permission training error](https://user-images.githubusercontent.com/47032409/124313856-1ab7ee00-db8f-11eb-9617-b5fa88ed642a.jpg)

**Reply by polm** [reply]
So the code here is just trying to make a directory under your current directory to save the model in. I can't imagine you'd need administrator privileges for that. Maybe the issue is the training1 directory doesn't exist yet? If you run `mkdir training1` before running training does that fix it?

Also please paste errors as text, not as screenshots.

**Reply by palitr** [reply]
Hi polm,

Thanks for your reply. The training1 directory does exist.

However I found what the issue was. The windows sync setting was on. Everytime the model was creating a new file, windows was trying to sync it to one drive and that wsa causing some locks. After I paused the sync, the process ran fine.

**Reply by polm** [reply]
Glad to hear you resolved the issue, thanks for letting us know!

**Reply by a-leena** [reply]
THANK YOU SO MUCH! COMPLETE LIFESAVER!!
